api/specs.proto

```diff
@@ -174,6 +177,21 @@ message ContainerSpec {
 	PullOptions pull_options = 10;
 }
 
+// ContainerHostConfigSpec specifies hostconfig for container.
+message ContainerHostConfigSpec {
```

Comment:
We really haven't preserved the split the between config and host config in SwarmKit.